Source/Find_Number_follow_Kaprekar_Constant.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
+#include <vector>
+#include <climits>
+using std::swap;
+using std::vector;
+using std::cout;
+using namespace std;
+
```

Comment:
This file is not following SRP, it contains all the kind of different methods which are not related to Kaprekar Constants